package.json

```diff
@@ -28,7 +28,6 @@
     "generate": "nx generate",
     "env-secrets": "./scripts/secrets.sh",
     "format": "nx format:write",
-    "format:write": "nx format:write",
```

Comment:
Does the sort script also format at the same time?

Reply:
This line is just removing a redundant script which I don't see used anywhere in the repo.

But yeah, the sort script runs as part of prettier, and calls the standard prettier json formatter.